specs/drafts/epics/model-catalog-planner--suitability-scorer.md

```
@@ -0,0 +1,81 @@
+
+  /** Record a TPS observation for a provider+model pair. */
+  recordTps(providerId: string, modelId: string, tps: number): void;
+
+  /** Record a request outcome (success/failure) for a provider+model pair. */
+  recordOutcome(providerId: string, modelId: string, success: boolean): void;
+}
+```
+
+### Scoring algorithm (v1)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)